public/app/features/inspector/InspectJSONTab.tsx

```diff
@@ -105,8 +105,12 @@ export class InspectJSONTab extends PureComponent<Props, State> {
           appEvents.emit(AppEvents.alertError, ['Unable to apply']);
         } else {
           const updates = JSON.parse(this.state.text);
+          const shouldUpdateGridPositionLayout = JSON.stringify(updates?.gridPos) !== JSON.stringify(panel?.gridPos);
```

Comment:
nit: Probably not an issue here but you can use `safeParseJson` and `safeStringifyValue` to be on the safe side here.
https://github.com/grafana/grafana/blob/36925ef5216e0de7f2c09c4f9540442095e0bf5e/public/app/core/utils/explore.ts#L183
https://github.com/grafana/grafana/blob/36925ef5216e0de7f2c09c4f9540442095e0bf5e/public/app/core/utils/explore.ts#L195

Reply:
is not lodash isEqual a better fit than comparing json strings?

Reply:
@torkelo yes, is a better fit, I  was a bit unsure to use lodash (_I remember the conversation about not using lodash, we still need to define which ones we want to keep using._) I will change it here to `isEqual` :), btw @torkelo do you having any feedback regarding the approach?